scripts/package-windows-images

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -ex
+
+cd $(dirname $0)/..
+
+source ./scripts/version.sh
+
+mkdir -p dist/artifacts
+
+# 1809
+crane pull --platform windows/amd64 ${REGISTRY}/${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-windows-amd64 rke2-windows-1809-images.tar.gz
```

Comment:
is crane the only way to do this? It'd be nice if we could use the same tooling for exporting all the images; if you're going to use crane for Windows we might as well use it for linux as well.

Reply:
Yes, crane is the only way we can pull windows images on linux drone hosts. I was actually investigating how crane could replace all of the existing docker cli usage for non-windows builds and simplify the build-images and package-images process. 

This change was required due to the need for Windows airgap support and discussion that took place on the best possible way to achieve it.

Reply:
I opened a new issue for tracking migrating to crane https://github.com/rancher/rke2/issues/1243